### ISSUE TITLE
but why

### ISSUE THREAD

**Issue by TheMemeSniper** [3kh0/sh1mmer.me#1]

**Comment by TheMemeSniper** [comment]
you do know the site's still up, right?

**Comment by RuralAnemone** [comment]
(*and, the shims aren't on the main*)

**Comment by 3kh0** [comment]
just incase you try to take it down

**Comment by TheMemeSniper** [comment]
mf it's open source??? us taking it down won't do shit

quit star whoring

**Comment by Unciaur** [comment]
Jesus christ people are dense lol

**Comment by 3kh0** [comment]
i have no clue how and why this is first on google

**Comment by Scaratek** [comment]
yall stupid. not talking about kait im talking about you retards

**Comment by 3kh0** [comment]
@Scaratek who tf asked, like you come in here acting like you know shit but you cant do shit, what is your income poor looking ass.

**Comment by Scaratek** [comment]
mfr i am in mercury. 
you cant talk shit either your banned for TN and your a massive fucking skid

**Comment by 3kh0** [comment]
@Scaratek lol, you silly ass cant prove im skid, lup cant, you guys only have one thing, only that.

**Comment by RuralAnemone** [comment]
would you like me to prove it

**Comment by RuralAnemone** [comment]
ok so this isn't going to be dmca'd, that's a joke, it's literally not illegal to fork repos
and you really aren't a skid by its formal definition (script kiddie, i.e. "an unskilled individual who uses scripts or programs developed by others") because I think I've seen you write your own code once

but

well, it really boils down to you having no idea how to use licenses and thereby unintentionally claiming others' projects as your own

for example, https://github.com/3kh0/ext-remover/discussions/165:
"You have attached a license to these tools - the WTFPL. You are not the rights-holder to these tools, which do not have the WTFPL attached to them."

...and you give off the general idea of you having created these things (LTBEEF, sh1mmer, etc) when you clearly haven't
and innocent children think that you are a master programmer
and the have no idea how much work went into it, etc, now they think coding is just smacking a keyboard until it works because surely nobody can understand _that_, etc, etc, there is no difference in their minds between malbolge and js, etc, etc, etc

I kinda just went on a rant there sorry

I also want to let you know that I still absolutely respect you as a person (which I hope everyone else can agree with)
but not really as a programmer

that's all :)

eptitiously,
-Rural

**Comment by RuralAnemone** [comment]
tl;dr:
why are you reading the tl;dr? read the original first!!!
please, for the good of us all, do your own work and give other people the credit they deserve
and also please don't think that I don't still respect your humanity or anything

**Comment by RuralAnemone** [comment]
also

my opinion doesn't really count in TN (or any other enterprise bypassing network of that sort)
so take this _cumo grano salis_

**Comment by 3kh0** [comment]
Finally, some who know the actual definition of skid, but for real, I agree

**Comment by RuralAnemone** [comment]
"skid" can be applied to the definition "uses others' code without consent/proper acknowledgement" so uh

I assume by "for real, I agree" you mean you agree with me being emptitious?

actually I'll just let you answer lol

**Comment by 3kh0** [comment]
Yes i do.

**Comment by RuralAnemone** [comment]
:)
